src/ypspur_ros.cpp

```diff
@@ -191,21 +191,33 @@ class YpspurRosNode
 #if !(YPSPUR_JOINT_ANG_VEL_SUPPORT)
     ROS_ERROR("JointTrajectory command is not available on this YP-Spur version");
 #endif
+    const ros::Time now = ros::Time::now();
+
     std_msgs::Header header = msg->header;
     if (header.stamp == ros::Time(0))
```

Comment:
Is it correct to overwrite a timestamp?
I think it is more safe to take a timestamp.

Reply:
Sorry, referenced line number is wrong. I comment about line 197 in new file.

Reply:
In ROS timestamp convention (especially in TF), zero timestamp means the latest possible time.
This is overwritten only if given timestamp is 0.

Reply:
OK, I understand the thought.